tools/chainsync-loadtest/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "chainsync-loadtest"
```

Comment:
please also add README.md file with info about this tool and examples on how to run it.

Reply:
done